packages/rx_bloc_cli/example/README.md

```diff
@@ -19,8 +19,9 @@
 15. [Patrol integration tests](#patrol-integration-tests)
 16. [Realtime communication](#realtime-communication)
 17. [Feature OTP](#feature-otp) 
-18. [CI/CD](#cicd)
-19. [Next Steps](#next-steps)
+18. [CI/CD](#cicd) 
+19. [Pin Code](#pin-code)
```

Comment:
This hyperlink won't work, this is leading to the pin-code instead of the feature-pin-code

Reply:
This was fixed